File: trail1/A 부터 B 까지/a-to-b.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a,b;
    cin>>a>>b;

    int i=a;

    while(i<=b){
        if(i%2==1){
            
            cout<<i<<" ";
            i*=2;
        }
        else{
            
            cout<<i<<" ";
            i+=3;
        }
    }
    return 0;
}

File: trail1/A 부터 B 까지/a-to-b.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner scin=new Scanner(System.in);

        int a=scin.nextInt();
        int b=scin.nextInt();

        for(int i=a;i<=b;){

            System.out.print(i+" ");
            if(i%2==1){
                i*=2;
            }
            else{
                i+=3;
            }
        }
    }
}